README.md

```diff
@@ -1,3 +1,273 @@
\ No newline at end of file
+Gardener uses Kubernetes to manage Kubernetes clusters. This documentation describes how to install Gardener on an existing Kubernetes cluster of your IaaS provider.
+> Where reference is made in this document to the *base cluster*, we are actually referring to the existing cluster where you will install Gardener. This helps to distinguish them from the clusters that you will create after the installation using Gardener. Once it's installed, it is also referred to as *garden cluster*. Whenever you create clusters, Gardener will create *seed clusters* and *shoot clusters*. In this documentation we will only cover the installation of clusters in one region of one IaaS provider. More information: [Architecture](https://gardener.cloud/030-architecture/).
+
+## Prerequisites
+
+* The installation was tested on Linux and MacOS
+* You need to have the following tools installed:
+  * [Docker](https://docs.docker.com/install/)
+  * [git](https://git-scm.com/downloads)
+  * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* You need a *base cluster*. Currently, the installation tools supports to install Gardener on the following Kubernetes clusters:
+  * [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster) on Google Cloud Platform (GCP)
+  * [Elastic Container Service for Kubernetes (EKS)](https://docs.aws.amazon.com/eks/) or [Kubernetes Operations (kops)](https://github.com/kubernetes/kops) on Amazon Web Services (AWS)
+  * [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/) on Microsoft Azure
+* Your base cluster needs at least 4 nodes with a size of 8GB for each node
+* You need a service account for the virtual machine instance of your IaaS provider where your Kubernetes version runs
+* You need to have permissions to access your base cluster's private key
+* You are connected to your Kubernetes cluster (environment variable `KUBECONFIG` is set)
+  * [Viewing kubeconfig (GKE)](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#viewing_kubeconfig)
+  * [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html)
+  * [Use Azure role-based access controls to define access to the Kubernetes configuration file in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/control-kubeconfig-access)
+
+## Procedure
+
+1. Clone the `sow` repository and build a Docker image that contains the command line tool [sow](https://github.com/gardener/sow) and all tools `sow` requires. Once the image is built and you have added the path to our [wrapper script](https://github.com/gardener/sow/blame/master/docker/bin/sow) to your `PATH` variable you can call `sow` on the command line.
+
+    ```bash
+    # setup for calling sow via the wrapper
+    git clone "https://github.com/gardener/sow"
+    cd sow
+    make # will build and tag the image
```

Comment:
The user needs to be in the Docker group or one has to run `sudo make` here.

Reply:
We now push the image to a registry, so the `make` step has become optional. Should still be mentioned in case someone needs to build the image manually for whatever reason.

Reply:
Then we should mention the registry. Which one is it?

Reply:
I rewrote this part referring to  eu.gcr.io/gardener-project/sow